presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/ColumnStatistics.java

```diff
@@ -31,6 +32,7 @@ public ColumnStatistics(
             DoubleStatistics doubleStatistics,
             StringStatistics stringStatistics,
             DateStatistics dateStatistics,
+            IntegerStatistics timestampStatistics,
```

Comment:
Initially we used a new `TimestampStatistics` class but later we thought of reusing `IntegerStatistics` since both have same functionality.